### PR TITLE
feat(i3s): load complete material texture sets

### PR DESCRIPTION
**Pull request by ibgreen**
## Goals

- Complete tranche 3 of the I3S supremacy roadmap: material fidelity for PBR texture sets.
- Preserve compatibility with existing single-texture consumers while exposing complete material resources.

## Changes

- Resolve every texture set referenced by an I3S material: base color, metallic-roughness, normal, occlusion, and emissive.
- Fetch/decode each selected texture resource independently and expose decoded resources through `content.textures`.
- Attach each decoded resource to its matching PBR material slot while retaining the existing `content.texture` behavior.
- Carry legacy I3S `none`, `repeat`, and `mirror` wrap declarations into material texture metadata.
- Add focused multi-texture and wrap tests and document the updated material-fidelity boundaries.

This is a dependent follow-up to #3805 and should merge after the tranche 2 branch.

## Validation

- `yarn lint fix`
- I3S TypeScript check (`node node_modules/typescript/bin/tsc -p modules/i3s/tsconfig.json --noEmit`)
- `yarn build`
- `yarn build-workers`
- `yarn test-node` (361 passed, 6 skipped)
- `yarn test-headless` (2,937 passed, 40 skipped)
- `yarn test-audit` (594 files, 0 violations)
- `cd website && yarn build` (successful; existing warnings/broken links reported)

### CODE REVIEW

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 1]
```diff
@@ -88,72 +88,43 @@ export async function parseI3STileContent(
...
-              imageLoaderOptions,
-              nestedContext
-            );
-            content.texture = getImageData(parsedTexture);
+  const textureResources = tileOptions.textureUrls?.length
+    ? tileOptions.textureUrls
+    : tileOptions.textureUrl
+      ? [
+          {
+            textureSetDefinitionId: 0,
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts]: Fixed in e171d9ae03: the legacy singleton texture URL now uses the first texture-set definition referenced by the material, preserving IDs such as 1 and allowing the texture to attach correctly.

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 2]
```diff
@@ -648,6 +696,30 @@ function setMaterialTexture(material, image: TileContentTexture): void {
...
+    material.occlusionTexture,
+    material.emissiveTexture
+  ];
+  for (const textureSlot of textureSlots) {
+    if (!textureSlot) {
+      continue;
+    }
+    const texture = textures[textureSlot.textureSetDefinitionId];
+    if (texture) {
+      textureSlot.texture = {source: {image: texture}};
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts:738]: Fixed in e171d9ae03: makePbrMaterial now clones every material texture slot, including nested PBR slots, before attaching per-tile images so shared definitions are never mutated across tiles.
